src/view/draggable/connected-draggable.js

```diff
@@ -258,10 +259,23 @@ const mapDispatchToProps: DispatchProps = {
   dropAnimationFinished: dropAnimationFinishedAction,
 };
 
+const defaultProps = ({
+  isDragDisabled: false,
+  // cannot drag interactive elements by default
+  disableInteractiveElementBlocking: false,
+}: DefaultProps);
+
+// eslint-disable-next-line
+/*::
```

Comment:
One last thing: Can you please add a comment explaining this in the code itself?